### DISCUSSION TITLE
RabbitMQ server is started but not able to access localhost url

### DISCUSSION THREAD

**Discussion by Vivek-S-Lumen** [rabbitmq/rabbitmq-server#7815] [Other] [answered]
Hi ,
I have installed RabbitMq server version 3.8.2 on my windows 11. It is showing running status in services but not able to use localhost url. where I have enable the rabbitmq-management plugin also. Still the problem is not resolved. any one please help on this.

FYI : I'm using windows 11 enterprise version with a build number of 22h2

**Comment by michaelklishin** [comment]
There isn't enough details to suggest much.

Use

```
rabbitmq-plugins.bat list
```

to see what plugins are enabled.

See [Troubleshooting Networking](https://rabbitmq.com/troubleshooting-networking.html) and Windows firewall settings to see if [the ports RabbitMQ uses](https://rabbitmq.com/networking.html#ports) are whitelisted or not.

Also, keep in mind that [RabbitMQ 3.8 is out of support](https://rabbitmq.com/versions.html). Consider upgrading to 3.11.

**Reply by Vivek-S-Lumen** [reply]
![image](https://user-images.githubusercontent.com/129164424/229709613-7cbfc85b-3a7b-4bad-a62f-731d0c48d34b.png)
Here are the enabled plugins

**Reply by Vivek-S-Lumen** [reply]
@michaelklishin did you have the time to check these in

**Reply by michaelklishin** [reply]
All this tells us that the plugin is enabled. This does not have any details on Windows firewall settings or any other observations recommended [earlier](https://rabbitmq.com/troubleshooting-networking.html).

Please use `3.11.x` and not `3.8.2`, a release that's 33 patches behind in the no longer supported 3.8 series.

**Reply by Vivek-S-Lumen** [reply]
@michaelklishin  I have tried installing 3.11.x as you mentioned in my system. my local machine version is windows 11 enterprise and having the build of 22H2. Even though the rabbitMQ server is not showing up in the localhost:15672. I checked with the my team whether any firewall is blocking the startup but we didn't seem any issue from the fire wall. Do you know what will the problem. If possible will try to connect and your help is much appreciated.

**Reply by michaelklishin** [reply]
There can be any number of issues, in fact, there is a separate guide on issues specific to Windows: https://rabbitmq.com/windows-quirks.html

**Comment by Vivek-S-Lumen** [comment]
<img width="905" alt="Screenshot 2023-04-06 132608" src="https://user-images.githubusercontent.com/129164424/230312442-16d05db2-c36e-4644-af68-3c6cdafa558c.png">
I have run the diagnostics. the above is the issue it is showing.

**Reply by lukebakken** [reply, marked as answer]
RabbitMQ is not running at all. You are still using version 3.8.2 as well.

I suggest un-installing RabbitMQ and Erlang, and starting again. Ensure that `%APPDATA%\RabbitMQ` is deleted after un-installation.
